.github/workflows/rawls-run-azure-e2e-tests.yaml

```diff
@@ -96,16 +100,16 @@ jobs:
       contents: 'read'
       id-token: 'write'
     outputs:
-      custom-version-json: ${{ inputs.build-branch && steps.render-rawls-version.outputs.custom-version-json || ''}}
+      custom-version-json: ${{ (needs.init-github-context.outputs.build-branch == true || needs.init-github-context.outputs.build-branch == 'true') && steps.render-rawls-version.outputs.custom-version-json || ''}}
```

Comment:
This is really annoying, but even though these arguments are defined as booleans, we end up with strings. And then 'false' is truthy, so just checking `needs.init-github-context.outputs.build-branch` leads to an incorrect condition.

I probably can ONLY check for 'true', but just in case I decided to check for both (which some folks on github issues recommend).

Reply:
Decided to switch to an explicit choice because the booleans still weren't working properly.